tests/dummy/app/templates/tooltip-auto-close.hbs

```diff
@@ -0,0 +1,50 @@
+<h2>Automatically Closing Tooltips</h2>
+
+<ul class="examples">
+
+  <li>
+    {{#test-component
+      tooltipContent='This shows on a click event, then hides'
+      tooltipEvent='click'
+      data-test='auto-close-basic'
+      tooltipDuration=2000
+    }}
+      Click me
+    {{/test-component}}
+  </li>
+
+  <li>
+    {{#test-component data-test='auto-close-component'}}
+      {{#tooltip-on-parent duration="1000" event="mouseenter"}}
```

Comment:
Sorry to be a stickler again but can you use single quotes for all the HTMLBars attributes?